docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -365,6 +365,10 @@ children:
     url: Loopback-component-authorization.html
     output: 'web, pdf'
 
+  - title: 'JWT Authentication'
```

Comment:
I think it's better to have
[LoopBack-component-authentication](https://github.com/strongloop/loopback-next/blob/master/docs/site/Loopback-component-authentication.md)
or 
[Authentication-overview](https://github.com/strongloop/loopback-next/blob/master/docs/site/Authentication-overview.md)
here.